javascript/utils/idp.js

```diff
@@ -29,6 +29,14 @@ firebaseui.auth.idp.isSupportedProvider = function(providerId) {
 
 
 /**
+ * The provider ID constant for the email auth provider
+ *
+ * @const {string}
+ */
+firebaseui.auth.idp.EMAIL_PROVIDER_ID = 'password';
```

Comment:
Replied the same comment to @TMSCH [here](https://github.com/firebase/firebaseui-web/pull/108/files/3cdef074eb095b6d5ce77bf59c1b9f0f0eb4c2e5#r105019960). Any ideas how to fix this?